internal/store/poddisruptionbudget.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package store
```

Comment:
It seems like we are using a field that does not exist anymore this most likely happened due to a bump, maybe there is something in their release file:

> type "k8s.io/api/policy/v1beta1".PodDisruptionBudgetStatus has no field or method PodDisruptionsAllowed

https://github.com/kubernetes/kube-state-metrics/blob/1f54b012d86766f9b4c45e8a743775e69515705f/internal/store/poddisruptionbudget.go#L90

Let me know if that makes sense, otherwise I can also take over, no problem! :)

Reply:
yes, I had understood the error (sorry for being not clear), I was not able to figure out what changes to make to fix it.
found the solution, so PodDisruptionsAllowed was renamed to DisruptionsAllowed in this PR: https://github.com/kubernetes/kubernetes/pull/85863